tests/datetime/test_dates.py

```diff
@@ -17,7 +17,10 @@
 
 from __future__ import division, print_function, absolute_import
 
+from datetime import date, datetime
```

Comment:
Maybe use `import datetime as dt` here, both for consistency with the strategy definitions and to avoid having `date/dates` and `datetime/datetimes` floating around in the same file?